<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workspace</class>
 <widget class="QMainWindow" name="workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>971</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);

font: 87 9pt &quot;Raleway Black&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>GESTION DES SALLES</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_R">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PDF">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convertir en PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cap">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure d'ouverture:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rech">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>437</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure de fermeture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Capacit√© de salle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Liste_Materiel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>ID Salle:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Code QR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;Sitka Display&quot;;

</string>
      </property>
      <property name="text">
       <string>Listes des salles</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_S">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Arial&quot;;
</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_ho">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_hf">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="lineEdit_lm">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_id">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>440</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>PAR IDSALLE</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_cap">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>480</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>PAR CAPSALLE</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_ho">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>520</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>PAR HOSALLE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>440</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TRIER:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EXCEL">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convertir en EXCEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imp">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="QRCODE_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>400</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSTAT">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>STAT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonplay">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PLAY VID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonstop">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>520</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>STOP VID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MODIFCATION</string>
     </attribute>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>ID Salle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Liste_Materiel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure d'ouverture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Heure de fermeture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>Capacit√© de salle:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cap_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 8pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_hf_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_ho_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="lineEdit_lm_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>310</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mail</string>
     </attribute>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paswd">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>255</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>325</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>325</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Smtp-server:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>185</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>465</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="server">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>150</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Server port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>360</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uname">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>220</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>mootez.slim@esprit.tn</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>360</y>
        <width>188</width>
        <height>85</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipant :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>290</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 7pt &quot;Raleway Black&quot;;</string>
      </property>
      <property name="text">
       <string>mootez.slim@esprit.tn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
font: 87 6pt &quot;Raleway Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
